Getting-Started.md

```diff
@@ -0,0 +1,145 @@
+2. Select Group
+3. Select Song
+4. Select Steps (usually Expert steps)
+5. Choose "Edit Existing"
+
+This is the editor view. Use the arrow keys to scroll through the chart. You can use the space bar to select the bounds on a range, and then use the `p` key to play the song at that range. 
+Turn off the measure lines in `Esc > Preferences > Show Measure Lines`. Make sure to press `enter` to save your changes.
+
+Open up `lua/mods.lua` in the text editor.
+Under the line `-- your code goes here here:`, add this:
+```lua
+setdefault {2, 'xmod', 100, 'overhead', 100, 'dizzyholds', 100, 'modtimer'}
+```
+This sets the scroll speed to `2x`, sets the perspective to "`overhead`", and does a couple of other things.
+The `setdefault` function takes in pairs of numbers and mods, and sets the mod to that amount.
+More information about `setdefault` can be found on [its documentation page](docs/setdefault.md).
+
+<a name="setting-mods"></a>
+
+## Setting Mods
+Now that you've set some base mods, you can now schedule mods to change at different beats of the song. To do that, you can use the [set function](docs/set.md).
+The `set` function works just like `setdefault`, except for an extra beat number at the beginning.
+Try choosing a mod from [the list](docs/mods.md), and applying it with set.
+Here's an example that turns `invert` on at beat 4, and turns it back off at beat 8.
+```lua
+-- on beat 4, set 100% invert
+set {4, 100, 'invert'}
+
+-- on beat 8, set 0% invert
+set {8, 0, 'invert'}
+```
+This example used invert, but `set` works with any mod. You can try changing out `invert` for another mod from [the list](docs/mods.md), or find more information can be found on [`set`'s documentation page](docs/set.md).
+
+<a name="smooth-transitions"></a>
+
+## Smooth Transitions
+If you tried the previous example, you'll notice that there's no animations; the mods instantly turn on and off. Sometimes that's okay, but lots of the time, you'll want to choose an animation to use.
+That's where the [ease function](docs/ease.md) comes in.
+The ease function works like set, except it needs two more arguments: a length, and an ease function.
+```lua
+-- ease {beat, length, ease, percent, mod}
+```
+Here's an example that that turns `invert` on at beat 12, and turns it back off at beat 16 smoothly.
+```lua
+-- on beat 12, for 2 beats, use the `outExpo` animation to set 100% invert
+ease {12, 2, outExpo, 100, 'invert'}
+
+-- on beat 16, for 2 beats, use the `outExpo` animation to set 0% invert
+ease {16, 2, outExpo, 0, 'invert'}
+```
+This example used a length of `2`, and the `outExpo` ease, but you can try changing the ease to another one from [the ease list](docs/eases.md), and you can change the length.
+You can find more information about `ease` on [its documentation page](docs/ease.md).
+
+<a name="conclusion"></a>
+
+## Conclusion
+Now you have everything you need to begin modding. The [main page](index.md) has links to other functions you can read about.
```

Comment:
I really really hope you didn't edit this file and move it in the same commit. I will just assume that this is the same text as before.

Reply:
😬

Reply:
will revert this change